app/views/settings/_my_applications.slim

```diff
@@ -38,11 +38,12 @@
                     code = application.redirect_uri
                   td
                     .input-group
-                      input.form-control id="client-id-#{application.uid}" type='text' value=application.uid readonly=1
+                      input.form-control id="client-id-#{application.uid}" type='text' value=application.uid readonly=1 aria-label='Client id'
```

Comment:
Use "ID" here; normally it wouldn't be a big deal but I think screen readers will pronounce this "id" and we want it to be pronounced "aye dee"

Reply:
It's funny because I put URL for this exact reason, but then just blanked on these haha.